_posts/en/newsletters/2020-02-12-newsletter.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Bitcoin Optech Newsletter #84'
+permalink: /en/newsletters/2020/02/12/
+name: 2020-02-12-newsletter
+slug: 2020-02-12-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter seeks help testing a Bitcoin Core release
+candidate and summarizes some discussion about the BIP119
+`OP_CHECKTEMPLATEVERIFY` proposal.  Also included is our regular section
+about notable code and documentation changes.
+
+## Action items
+
+- **Help test Bitcoin Core 0.19.1rc1:** this upcoming maintenance
+  [release][bitcoin core 0.19.1] includes several bug fixes.
+  Experienced users are encouraged to help test for any regressions or
+  other unexpected behavior.
+
+## News
+
+- **`OP_CHECKTEMPLATEVERIFY` (CTV) Workshop:** video ([morning][ctv
+  morning vid], [afternoon][ctv afternoon vid]) and a [transcript][ctv
+  transcript] are available from a recent workshop about [BIP119][] CTV.
+  If this proposed soft fork is adopted, users would be able to use a
+  new CTV opcode to create [covenants][topic covenants] with less
+  interaction than would be required using current consensus rules.
+  Several possible applications of the opcode were discussed, with most
+  attention being focused on vaults and compressed payment batching
+  (sometimes called *congestion control transactions*).  A significant
```

Comment:
Should we link to previous newsletter writing about congestion control?

https://bitcoinops.org/en/newsletters/2019/05/29/#proposed-transaction-output-commitments